### PR TITLE
Fixes to python test environment

### PR DESCRIPTION
**Pull request by AaronDavidSchneider**

### CODE REVIEW

File: gcm_toolkit/tests/test_interface.py [hunk 1]
```diff
@@ -43,15 +43,21 @@ def test_prt_interface(petitradtrans_testdata, all_raw_testdata):
...
+            len(dsi_clouds["lon"]),
+            len(dsi_clouds["Z"]),
+        )
+    )
+    dsi_clouds["ClAb"] = (("time", "lat", "lon", "Z"), tmp * 1e-5)
+    dsi_clouds["ClDs"] = (("time", "lat", "lon", "Z"), tmp * 1e2)
+    dsi_clouds["ClDr"] = (("time", "lat", "lon", "Z"), tmp * 1e3)
+    dsi_clouds["ClVf_Test1[s]"] = (("time", "lat", "lon", "Z"), tmp * 0.5)
+    dsi_clouds["ClVf_Test2[s]"] = (("time", "lat", "lon", "Z"), tmp * 0.5)
+    tools._replace_model("0", dsi_clouds)
```
Comment (AaronDavidSchneider) [inline gcm_toolkit/tests/test_interface.py:60]: @Kiefersv, could you comment on these changes I had to make? I think the Z_l here is strange?!

Comment (Kiefersv) [reply gcm_toolkit/tests/test_interface.py:60]: I double checked and you are indeed right that it should be Z. No idea why it worked before with Z_l.

Comment (AaronDavidSchneider) [reply gcm_toolkit/tests/test_interface.py:60]: No. Its likely problems related to xarray's master branch. But this looked odd to me.

**Review by Kiefersv — APPROVED** [review]

**Review by Kiefersv — COMMENTED** [review]
Changes look fine, just the xarray test that still fails.

**Pull request merged**